apisix/init.lua

```diff
@@ -313,7 +314,13 @@ end
 
 
 local function verify_tls_client(ctx)
-    if ctx and ctx.ssl_client_verified then
+    local matched = router.router_ssl.match_and_set(ctx, true)
+    if not matched then
+        return true
```

Comment:
little confused, we should return false, all right?

Reply:
Return true means the verification is successful when this function is called with non-TLS request.

Reply:
got it, many thx